kong-3.1.0-0.rockspec

```diff
@@ -94,6 +94,7 @@ build = {
     ["kong.resty.dns.client"] = "kong/resty/dns/client.lua",
     ["kong.resty.dns.utils"] = "kong/resty/dns/utils.lua",
     ["kong.resty.ctx"] = "kong/resty/ctx.lua",
+    ["kong.resty.websocket.client"] = "kong/resty/websocket/client.lua",
```

Comment:
We need to move this into a patch file, please do not copy this entire file into the Kong repo.

Reply:
I think it will be done in other PR after this merged.